### DISCUSSION TITLE
Is there a place to discuss development?

### DISCUSSION THREAD

**Discussion by willparsons** [jesseduffield/lazygit#4087] [General]
I'm getting stuck trying to contribute but I don't want to pollute the discussions/issues with noob questions about the codebase.

Is there a better place to discuss development? Or could we create a development category for the discussions to keep it out of the way?

**Comment by mark2185** [comment]
You can always come to [the discord channel](https://discord.gg/ehwFt2t4wt) (taken from [the Contributing section](https://github.com/jesseduffield/lazygit?tab=readme-ov-file#contributing) of the `README.md`).

If `discord` is not your thing, feel free to ask questions here.

**Reply by willparsons** [reply]
My bad, I looked for "lazygit discord" and couldn't find anything so I figured it didn't exist. Thanks.

**Comment by stefanhaller** [comment]
And if your question is about that draft PR that you just opened, the best place to ask questions is right there, of course. (I was going to respond to it but didn't yet because I wasn't sure how draft it still is.)

**Reply by willparsons** [reply]
Ah yeah, the PR is very early.

I wasn't sure if anyone would see if I put them there plus I figured it might get a bit noisy since I'm not clear yet on implementing multiple things.

I'll keep it in mind, thank you.
